scrapers/bugzilla.py

```diff
@@ -0,0 +1,98 @@
+# SPDX-License-Identifier: GPL-3.0+
+from scrapers.base import BaseScraper
+from purview.models.bugzilla import BugzillaBug
+from purview.models.user import User
+import purview.utils.general as utils
+from purview import log
+
+
+class BugzillaScraper(BaseScraper):
+
+    def run(self, since=None):
+        log.info('Starting initial load for Bugzilla bugs')
+        # Initialize a start date from which all builds must be fetched
+        # If no input is given by the user, fetch builds from the past two years
+        if since is None:
+            since = self.default_since
+        else:
+            since = utils.timestamp_to_datetime(since)
+
+        bugs = self.get_bugzilla_bugs(since)
+        log.info('Successfully fetched {0} bugs from teiid'.format(len(bugs)))
+        self.update_neo4j(bugs)
+        log.info('Initial load of Bugzilla bugs complete!')
+
+    def get_bugzilla_bugs(self, start_date):
+        # SQL query to fetch all (Red Hat) bugs from start date until now
+        log.info('Getting all Bugzilla bugs since {0}'.format(start_date))
+        sql_query = """
```

Comment:
As discussed, you should be able to left join all the users here so you only need to do one SQL query.

Reply:
Done!